dockerfile-demo-DB

```diff
@@ -0,0 +1,92 @@
+# Base
+
+FROM debian:latest
+
+# Install 
+
+RUN apt-get update && \
+    apt-get install -q -y \
+    locales \
+    wget \
+    git \
+    openssh-server \
+    bzip2 \
+    curl \
+    libpq-dev \
+    nano \
+    sudo
+
+# Set LOCALE
+
+RUN echo "pt_BR.UTF-8 UTF-8" > /etc/locale.gen
+RUN locale-gen pt_BR.UTF-8
+RUN update-locale pt_BR.UTF-8
+ENV LC_ALL pt_BR.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# Instal repo git lfs
+
+#RUN echo 'deb http://http.debian.net/debian wheezy-backports main' > /etc/apt/sources.list.d/wheezy-backports-main.list
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+RUN apt-get install git-lfs
+
+# Install POSTGRESQL
+
+RUN apt-get update && \
+    apt-get install -q -y \
+    postgresql \
+    postgresql-contrib \
+    postgis
+
+# Create DENGUE USER
+
+RUN useradd -s /bin/bash -g root -G sudo --home=/home/dengue/ --create-home dengue
+RUN echo 'dengue:dengueadmin' | chpasswd
+RUN echo '%dengue  ALL=(ALL:ALL) ALL' >> /etc/sudoers
+USER dengue
+
+# Change the permissions for the user home directory
+RUN mkdir /home/dengue/logs
+WORKDIR /home/dengue
+
+#RUN pg_ctl -D /home/postgresql/9.6/bdata -l start
+RUN PATH=/usr/lib/postgresql/9.6/bin:$PATH
+RUN /usr/lib/postgresql/9.6/bin/initdb -E utf8 -D /home/dengue/bdata 
+
+USER root
+# Adjust PostgreSQL configuration so that remote connections to the database are possible.
+RUN echo "host     all             all             0.0.0.0/0               md5" >> /etc/postgresql/9.6/main/pg_hba.conf
+# And add ``listen_addresses`` to ``postgresql.conf``
+RUN sed -i -e"s/^#listen_addresses =.*$/listen_addresses = '*'/" /etc/postgresql/9.6/main/pg_hba.conf
+EXPOSE 5432
+
+# install MINICONDA
+
+RUN wget -O ~/Miniconda.sh http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+RUN cd ~ && \
+ bash Miniconda.sh -b -p /home/dengue/miniconda && \
+ rm Miniconda.sh
+
+#GIT clone alias DEVELOP
+RUN git clone -b develop https://github.com/AlertaDengue/AlertaDengue.git
+
+ENV PATH="/home/dengue/miniconda/bin:${PATH}"
+
+RUN /home/dengue/AlertaDengue/AlertaDengue/environment.yml /tmp/environment.yml
```

Comment:
these 2 RUN instruction (this line and below, about environment.yml and requirements-dev) seems to be wrong, probably you are trying to use ADD .. but also it is unnecessary  ... as you are cloning the repo inside the image .. you don't need to add any file from host.

Reply:
! Yes, in fact the two lines of RUN envirioment is residue of when we were doing the ADD of the local envirioment files.
I'll make a new PR ...